llvm/test/MachineVerifier/test_g_extract_subvector.mir

```diff
@@ -1,4 +1,6 @@
-# RUN: not --crash llc -o - -run-pass=none -verify-machineinstrs %s 2>&1 | FileCheck %s
+# RUN: not --crash llc -o - -run-pass=none -mtriple=arm64 -verify-machineinstrs %s 2>&1 | FileCheck %s
+# REQUIRES: aarch64-registered-target
```

Comment:
This shouldn't be in this patch, and is the rare case that shouldn't require a target

Reply:
I mentioned above that my llc complained about being unable to get a target.

Reply:
This is still unrelated to this PR and should be separate, but see my comment above about it's really REQUIRES: default_triple